### ISSUE TITLE
why my plotly data json is so large

### ISSUE THREAD

**Issue by LMerCy** [allegroai/trains#198]
I use plotly to draw pr curve, but i found the saved json in web is so large, about 11M. Here is my code:
```
for i in range(num_epoch):
   fig = make_subplots(
                    rows=1, cols=2, subplot_titles=('Precision x Recall curve {}'.format(distance[i]),
                                                    'Precision,Recall X score curve {}'.format(distance[i])),
                )
   fig.add_trace(go.Scatter(x=recalls[i], y=precisions[i], name='pr',mode='lines'),
                              row = 1, col = 1)

   fig.add_trace(go.Scatter(x=score, y=np.multiply(precisions[i], recalls[i]), name='Acc', mode='lines'),
                              row=1, col=2)
   self.logger.report_plotly(pr', iteration=0, figure=fig)
```
i would do this after each training epoch. Does this add the new pr data to old data after each epoch, how can i just replace the old data?

**Comment by LMerCy** [comment]
Thans to @bmartinn , i solve the problem by round down numbers, and the plot size decrease to 300Kb from 11M

**Comment by bmartinn** [comment]
@LMerCy nice!

BTW: if you use [report_scatter2d[(https://allegro.ai/docs/examples/reporting/scatter_hist_confusion_mat_reporting/#2d-scatter-plots) the sub-sampling is built into the call, so you do not end up with plots that are too large to display.

Out of curiosity, how many digits did you leave? My thinking is to try and add this as a parameter to to `report_plotly` making it easier for users to control the plotly data being sent.

**Comment by bmartinn** [comment]
@LMerCy updating here that with the latest RC `0.16.2rc0` you can now limit the number of digits for all plot reporting.
See example [here](https://github.com/allegroai/trains/blob/10ec4d56fb4a1f933128b35d68c727189310aae8/docs/trains.conf#L40)

**Comment by LMerCy** [comment]
@bmartinn Thanks, great!

**Comment by bmartinn** [comment]
Closing issue, fix was deployed in release `trains` `0.16.2`
